src/app.js

```diff
@@ -1 +1,95 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+
+function createULElementWithId(){
+    const unorderedList = document.createElement('ul');
+    unorderedList.setAttribute('id', 'countryList');
+    document.body.appendChild(unorderedList);
+    return unorderedList;
+}
+
+function setContinentColor(continentName){
+    switch(continentName){
+        case 'Africa':
+            return 'blue';
+            break;
+        case 'Americas':
+            return 'green';
+            break;
+        case 'Asia':
+            return 'red';
+            break;
+        case 'Europe':
+            return 'yellow';
+            break;
+        case 'Oceania':
+            return 'purple';
+            break;
+        default:
+            return 'gray';
+    }
+}
+
+async function fetchCountries(){
+    try{
+        //haal alle landen op via rest api restcountries
+        const result = await axios.get('https://restcountries.com/v2/all');
+
+        //creeer een unordered list om de landen in te plaatsen
+        createULElementWithId();
+
+        //maak een nieuwe array met de landnamen
+        const country = result.data.map((country) => {
+            return country;
+        });
```

Comment:
Als je per se een nieuwe array wilt dan kun je beter zoiets doen:
```suggestion
        const country = [ ...result.data ];
```